javascript/javascript3/week2/currency-calculator.js

```diff
@@ -0,0 +1,84 @@
+const BASE_URL = "https://v6.exchangerate-api.com";
+const MY_KEY = "a7b1d475b4a2c818963b7f85";
+
+const currencies1 = {
+  EUR: "Euro",
+  USD: "United States Dollar",
+  DKK: "Danish Krone",
+  GBP: "British Pound",
+  JPY: "Japanese yen",
+  CNY: "Renminbi",
+  AUD: "Australian Dollar",
+  CAD: "Canadian Dollar",
+  CHF: "Swiss franc",
+  SEK: "Swedish krona",
+};
+const currencies2 = {
+  DKK: "Danish Krone",
+  EUR: "Euro",
+  USD: "United States Dollar",
+  DKK: "Danish Krone",
+  GBP: "British Pound",
+  JPY: "Japanese yen",
+  CNY: "Renminbi",
+  AUD: "Australian Dollar",
+  CAD: "Canadian Dollar",
+  CHF: "Swiss franc",
+  SEK: "Swedish krona",
+};
+
+const firstCurrency = document.getElementById("first-currency");
+const secondCurrency = document.getElementById("second-currency");
+
+//Add <option> inside the <select> from Const currencies1 and currencies2
+const getData = (data) => {
+  return Object.entries(data).map(([country, currency]) => {
+    console.log(country, currency);
+    return `<option value="${country}">${country} - ${currency}</option>`;
+  });
+};
+
+firstCurrency.innerHTML = getData(currencies1);
+secondCurrency.innerHTML = getData(currencies2);
+
+// Make the API request of the selected currency
+async function getCurrencies() {
+  const firstValue = firstCurrency.value;
+  const secondValue = secondCurrency.value;
+  const amount = document.getElementById("input-value").value;
+
+  await fetch(`${BASE_URL}/v6/${MY_KEY}/latest/` + firstValue)
+    .then((response) => {
+      return response.json();
+    })
+    .then((data) => {
+      showCurrency(data, firstValue, secondValue, amount);
+    })
+    .catch((error) => console.error("The error:", error.message));
```

Comment:
Nice that you use then and catch! Check also https://dmitripavlutin.com/javascript-fetch-async-await/, `Handling fetch errors` part.